README.md

```diff
@@ -10,15 +10,16 @@ Witnet repository containing:
 
 - Feel free to modify/remove/add source files at `./requests/*.js`, following instructions in [this tutorial](https://docs.witnet.io/tutorials/bitcoin-price-feed/sources/).
 
-- Convert your data request RADON scripts into actual Witnet Request contracts:
+- Convert your Witnet RADON scripts into actual Solidity contracts:
```

Comment:
All the explanations below this line are now apparently untrue.

Reply:
Right. I will overhaul the whole README file as last thing in this PR.